test/config/config.go

```diff
@@ -104,13 +109,17 @@ func BindFlags(set *flag.FlagSet) {
 
 	// runtime
 	set.StringVar(&env.servicesImageVersion, prefix+"services-image-version", infrastructure.GetRuntimeImageVersion(), "Set the services (jobs-service, data-index) image version")
+	set.StringVar(&env.servicesImageNamespace, prefix+"services-image-namespace", infrastructure.DefaultImageNamespace, "Set the services (jobs-service, data-index) image namespace")
+	set.StringVar(&env.servicesImageRegistry, prefix+"services-image-registry", infrastructure.DefaultImageRegistry, "Set the services (jobs-service, data-index) image registry")
 	set.StringVar(&env.dataIndexImageTag, prefix+"data-index-image-tag", "", "Set the Kogito Data Index image tag ('services-image-version' is ignored)")
 	set.StringVar(&env.jobsServiceImageTag, prefix+"jobs-service-image-tag", "", "Set the Kogito Jobs Service image tag ('services-image-version' is ignored)")
 	set.StringVar(&env.mgmtConsoleImageTag, prefix+"management-console-image-tag", "", "Set the Kogito Management Console image tag ('services-image-version' is ignored)")
 
 	// build
 	set.StringVar(&env.mavenMirrorURL, prefix+"maven-mirror-url", "", "Maven mirror url to be used when building app in the tests")
 	set.StringVar(&env.buildImageVersion, prefix+"build-image-version", infrastructure.GetRuntimeImageVersion(), "Set the build image version")
+	set.StringVar(&env.buildImageNamespace, prefix+"build-image-namespace", "", "Set the build image namespace")
```

Comment:
I find it confusing to have default values for services but not for build ...

Reply:
I did this way at first, but then I realise that with default values, we'll always be sending the image runtime and image S2I attributes, and then we will stop covering the ability of the operator to set these values.

Also, I didn't want to change how was done at first. With services, we always send the image values. That's why services have defaults and images not. 

What do you think about it? I still can change it..

Reply:
> With services, we always send the image values

what do you mean ?
In the code of tests, we always force the services image ?

> and then we will stop covering the ability of the operator to set these values.

This is the same for services in fact.

Indeed, we are setting most of the time the build and services.
But we should have same behavior for build and for services I think.

IMO, we should have default values for image version/namespace/registry and set only if given.
In that case, we can still execute the tests letting the operator set the default values.
wdyt ?

Reply:
As agreed, I removed the defaults in the services and images parameters (registry, namespace and version). If not set, the operator will deal with it using the defaults, so we can cover this functionality in the operator.